.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,99 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Weekly Monday 6AM build
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 6 * * 1'
+
+jobs:
+  pep:
+    runs-on: ubuntu-16.04
+    steps:
```

Comment:
Ubuntu 16.04 is the latest they have? Seems like 20.04 has been out a few months, I'd hope they'd at least have 18.04...

Reply:
For PEP, I guess we can use `ubuntu-latest`. Good catch!